check/pytest-and-incremental-coverage

```diff
@@ -19,7 +28,28 @@ cd ${own_directory}
 repo_dir=$(git rev-parse --show-toplevel)
 cd ${repo_dir}
 
-PYTHONPATH="$(pwd)" python dev_tools/run_pytest_and_incremental_coverage.py $@
+# Figure out which branch to compare against.
+rest="$@"
+if [ ! -z "$1" ] && [[ $1 != -* ]]; then
+    branch="$1"
+    rest="${@:2}"
+elif [ ! -z $(git branch --list upstream/master) ]; then
+    branch=upstream/master
+elif [ !-z $(git branch --list origin/master) ]; then
+    branch=origin/master
+elif [ !-z $(git branch --list master) ]; then
+    branch=master
+else
+    echo -e "\e[31mNo master branch found to compare against. Argument #1 must be what to diff against (e.g. 'origin/master' or 'HEAD~1').\e[0m"
+    exit 1
+fi
+set -e
+git rev-parse "${branch}" > /dev/null
+set +e
+echo "Comparing against revision '${branch}'."
+
+# Run it.
+PYTHONPATH="$(pwd)" python dev_tools/run_pytest_and_incremental_coverage.py "${branch}" "${rest}"
```

Comment:
The quotes need to be removed so $rest can be expanded.

Also, run_pytest_and_incremental_coverage.py seems to ignore all arguments after the first.

Reply:
Obsoleted.